posthog/api/capture.py

```diff
@@ -54,6 +54,7 @@ def log_event(data: Dict, event_name: str) -> None:
     # TODO: Handle Kafka being unavailable with exponential backoff retries
     try:
         KafkaProducer().produce(topic=KAFKA_EVENTS_PLUGIN_INGESTION_TOPIC, data=data)
+        statsd.incr("posthog_cloud_plugin_server_ingestion")
```

Comment:
Let's keep track of this one too, just name it as something else. I know we already keep track of dlq counts, but there could be something else in the middle here too that's a problem.

Reply:
confused - what do you want me to change here?

Reply:
sorry I commented in the wrong place - I meant keep the other one too that you removed. But now looking at code there's already the failure one captured there & this move doesn't really change anything we were running it just before `capture_internal` before and now we're running it in `log_event` (https://github.com/PostHog/posthog/blob/cf94730690c02c465241b6afbbf0830507d9c04a/posthog/api/capture.py#L300).